### PR TITLE
Components revamp

### PR DESCRIPTION
**Pull request by bluong**
Revamping the Components Tab.

Now there is some text at the top expressing our intent to treat our objects as components to place onto a table, and then listing each component with a label along with a summary snippet of how the component is generated via tooltip.

Feel free to iterate and criticize, especially for the text section/feel of the page.

screen shot 2015-01-08 at 6 16 06 pm

### CODE REVIEW

**Comment by crmorford** [comment]
I really like the text snippet you wrote in that screenshot! I'd consider for the last sentence, maybe saying "Below is a list of some of the most frequently used components used in Plottable tables" or something that makes it clear that it isn't an complete list.

**Comment by bluong** [comment]
Unless I'm mistaken, I hopefully listed every component in the codebase and I think it would be good to list every component potentially on this list, but possibly to be safe I'll update the note.

The only difference is specifically gridlines, since it can't be instantiated by itself...

**Comment by crmorford** [comment]
Ah okay, I still don't have this running on my computer, so I had the old list in my mind (missing stacked/clustered stuff). If it's all there, never mind!

**Comment by crmorford** [comment]
Actually looking at the site now :) I think it's much better than previous.
Feedback:
1. if the font in the tooltips could be made bigger, that would greatly increase readability
2. I know this is the components page...but I think we should consider having one full plot at the end, maybe with the table cells outlined? Just to reiterate the fact that the point of knowing components is so that you can stick them in a table with other things.

**Comment by crmorford** [comment]
Looking really good right now. A few typos, but I'm okay with merging after those are fixed.

**Comment by jtlan** [comment]
Can we capitalize "Component" whenever we're referring to a Plottable Component?

File: components/index.html [hunk 1]
```diff
@@ -2,90 +2,231 @@
...
+<style>
+.qtip-dark {
+  font-size: 20px;
+  max-width: 700px;
+}
+</style>
+
+<div class="container">
+  <h2>The Component List</h2>
+The best way to think about Plottable that you will not be rendering plots, but you will be rendering components or tables of components.  Even a plot is considered a component!<br><br>
```
Comment (jtlan) [inline components/index.html]: "will not be rendering *charts*"?

File: components/index.html [hunk 2]
```diff
@@ -2,90 +2,231 @@
...
+By thinking in this manner, one can imagine that any component can be dropped into any cell of a table, meaning you have flexibility on where components go in relation to other components.<br><br>
```
Comment (jtlan) [inline components/index.html]: "*you* can imagine" for consistency? (we use "you" above and "one" here)

**Comment by jtlan** [comment]
Looks pretty solid. Can you find a docs person and ask them to go over it with you?

**Comment by bluong** [comment]
New PR to retaret #62

**Pull request closed without merging**